src/v/kafka/server/handlers/incremental_alter_configs.cc

```diff
@@ -423,6 +431,7 @@ inline std::string_view map_config_name(std::string_view input) {
       .match("log.message.timestamp.type", "log_message_timestamp_type")
       .match("log.compression.type", "log_compression_type")
       .match("log.roll.ms", "log_segment_ms")
+      .match("log.cleaner.delete.retention.ms", "tombstone_retention_ms")
```

Comment:
title of PR talks about `delete.retention.ms`. Is that the same as `log.cleaner.delete.retention.ms`?

Reply:
[In Kafka, `delete.retention.ms` is the topic property, while `log.cleaner.delete.retention.ms` is the server default property.](https://docs.confluent.io/platform/current/installation/configuration/topic-configs.html#delete-retention-ms) This function, `map_config_name()`, is only used in [alter_broker_configuartion()](https://github.com/redpanda-data/redpanda/blob/97c11fea87c952ca9abc430eca26123d8d1c36e3/src/v/kafka/server/handlers/incremental_alter_configs.cc#L429) (side note: we have a typo here! will fix)